packages/core/src/storage/vectorStore/MilvusVectorStore.ts

```diff
@@ -71,13 +76,54 @@ export class MilvusVectorStore implements VectorStore {
     return this.milvusClient;
   }
 
-  public async connect(collection: string): Promise<void> {
+  public async createCollection() {
+    await this.milvusClient.createCollection({
+      collection_name: this.collection,
+      fields: [
+        {
+          name: this.idKey,
+          data_type: DataType.VarChar,
+          is_primary_key: true,
+          max_length: 200,
+        },
+        {
+          name: this.embeddingKey,
+          data_type: DataType.FloatVector,
+          dim: 1536,
+        },
+        {
+          name: this.contentKey,
+          data_type: DataType.VarChar,
+          max_length: 9000,
+        },
+        {
+          name: this.metadataKey,
+          data_type: DataType.JSON,
+        },
+      ],
+    });
+    await this.milvusClient.createIndex({
+      collection_name: this.collection,
+      field_name: this.embeddingKey,
+    });
+  }
+
+  public async connect(): Promise<void> {
```

Comment:
It's better DX if the user doesn't have to call this `connect` before using the store.
Instead the store can do this check itself.
To not add unnecessary latency, this check should be only done once (lazily)
```suggestion
  private async ensureCollection(): Promise<void> {
```